### PR TITLE
Testing.md: Add information what to put in Gemfile in this file instead of just referring to other file

### PR DESCRIPTION
**Pull request by janpio**
...

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @janpio</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Testing.md [hunk 1]
```diff
@@ -70,7 +70,11 @@ If you want to run code style verification only for one tool, use `bundle exec r
 
 After introducing some changes to the _fastlane_ source code, you probably want to test the changes for your application. The easiest way to do so it use [bundler](https://bundler.io/).
 
-Copy the `Gemfile` [.assets/Gemfile](.assets/Gemfile) from your local _fastlane_ clone and drop it into your project's root folder. If you already have a `Gemfile` in your project's root folder, adapt it to match the template.
+Adapt your `Gemfile` in your project's root folder and replace the `gem 'fastlane'` line with:
```
Comment (joshdholtz) [inline Testing.md]: Can we change `Adapt` to `Edit`? `Edit` feels like it goes better with `replace` later on in this line 🙃

**Review by joshdholtz — CHANGES_REQUESTED** [review]
I'm feeling very nit picky today 😬 Sorry!

File: Testing.md (hunk 1 above)
Comment (janpio) [reply Testing.md]: Sure.

I hereby give you permission to make nitpicky changes in my PRs yourself before merging, especially if they only concern one word 🍅

**Comment by joshdholtz** [comment]
@janpio Is there a way to re-run this appveyor build? I always look for it and can never figure it out 😛

**Comment by janpio** [comment]
@joshdholtz If you are logged in and part of the team by @KrauseFx:
image
But currently of not much use as f*ing chocolatey is timing out.

**Review by joshdholtz — APPROVED** [review]
Thanks for making that change! 👍

**Pull request merged**